lib/screens/app/ecosystem/overview.dart

```diff
@@ -37,6 +40,22 @@ class _OverviewState extends State<Overview> {
     ]);
   }
 
+  void onGet() {
+    String userAccount = SettingsNotifier.of(context).accountName;
+    UrlLauncher.launch("https://www.joinseeds.com/buy-seeds?acc=$userAccount", forceSafariVC: false, forceWebView: false);
+  }
+
+  void onDonate() {
+    NavigationService.of(context).navigateTo(
+      Routes.transferForm,
+      TransferFormArguments(
+        "balisupport.org",
+        "kaelaatleewo",
```

Comment:
move this to app config?

Reply:
all this data should be in app config and these values perhaps added as default values